### PR TITLE
DOCS-2424: Merge autogen: baserc, datamgr

### PR DESCRIPTION
**Pull request by andf-viam**
Merge autogen content for `base_remote_control` and `data_manager` service APIs.

### CODE REVIEW

File: static/include/services/apis/generated/data_manager.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+**Returns:**
+
+- [(map[string]interface{})](https://pkg.go.dev/builtin#string): The command response.
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+**Example:**
+
+```go {class="line-numbers linkable-line-numbers"}
+// This example shows using DoCommand with an arm component.
```
Comment (JessamyT) [inline static/include/services/apis/generated/data_manager.md:80]: Is there a realistic way in which we imagine someone could use DoCommand with the data service? It seems that, while we use arm component for all the component examples which is already a bit of a stretch, using an arm component as a stand-in for the data service is perhaps such a dissimilar example that it's unhelpful? Maybe something with generic service for all services? Also, are there any services that do not support Do?

File: static/include/services/apis/generated/data_manager.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+- `ctx` [(Context)](https://pkg.go.dev/context#Context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+
+**Returns:**
+
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+**Example:**
+
+```go {class="line-numbers linkable-line-numbers"}
+// This example shows using Close with an arm component.
```
Comment (JessamyT) [inline static/include/services/apis/generated/data_manager.md:110]: Same as above; can we make a service Close example? Maybe with generic service?

File: static/include/services/apis/generated/data_manager.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context#Context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `extra` [(map[string]interface{})](https://go.dev/blog/maps): Extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+**Example:**
```
Comment (JessamyT) [inline static/include/services/apis/generated/data_manager.md:23]: Nice-- "Example" isn't even on the current data page; hurray for automation creating better consistency and nice formatting!

File: static/include/services/apis/generated/base_remote_control.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+{{< tabs >}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- None.
+
+**Returns:**
+
+- [([]input.Control)](https://pkg.go.dev/go.viam.com/rdk/components/input#Control): A list of inputs from the controller that are being monitored for that control mode.
```
Comment (JessamyT) [inline static/include/services/apis/generated/base_remote_control.md:14]: Ah sweet this fixes what used to be an erroneous link to Python! Very nice.

File: static/include/services/apis/generated/base_remote_control.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+- `ctx` [(Context)](https://pkg.go.dev/context#Context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+
+**Returns:**
+
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+**Example:**
+
+```go {class="line-numbers linkable-line-numbers"}
+// This example shows using Close with an arm component.
```
Comment (JessamyT) [inline static/include/services/apis/generated/base_remote_control.md:103]: [opt] Same comment as with data service; can we at least use a service here? Existing example on this page is this:
```
// Close out of all remote control related systems.
err := baseRCService.Close(context.Background())
```
which seems nicer?

**Review by JessamyT — APPROVED** [review]
LGTM % would be great to make docommand and close etc. examples for services use a service instead of arm component.

File: static/include/services/apis/generated/data_manager.md (hunk 1 above)
Comment (andf-viam) [reply static/include/services/apis/generated/data_manager.md:80]: No, all resources that inherit from resource.* (in this case [resource.Resource](https://pkg.go.dev/go.viam.com/rdk@v0.30.0/resource#Resource)) must share the same code sample for those inherited methods.

No there are not.

**Pull request merged**